cgm/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+autoreconf -fi
+./configure --prefix=$PREFIX --enable-optimize --disable-debug --enable-shared
```

Comment:
CGM needs to be pointed to a solid modeling engine - typically Cubit - but we can't provide it on Conda/binstar